packages/cw1155/src/query.rs

```diff
@@ -27,6 +27,12 @@ pub enum Cw1155QueryMsg {
         start_after: Option<HumanAddr>,
         limit: Option<u32>,
     },
+    /// Query approved status `owner` granted to `operator`.
+    /// Return type: ApprovedForAllItemResponse
+    ApprovedForAllItem {
```

Comment:
I would rename to `IsApprovedForAll` that would suggest a boolean as a response.

The other one is a bit vague (calling it `ListApprovedForAll` would be more explicit), but I think clear enough.

Reply:
> I would rename to `IsApprovedForAll` that would suggest a boolean as a response.

that's better, thanks.

> The other one is a bit vague (calling it `ListApprovedForAll` would be more explicit), but I think clear enough.

I think current overall naming conversion is using nouns for message branch names, so I guess we can hold this one for now?